Assets/Scripts/Copilots/HealingBubble.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class HealingBubble : MonoBehaviour
+{
+    //variables for spore particle system
+    public float sporeTimer;
+    private bool sporeStart;
+    private ParticleSystem spores;
```

Comment:
Might just be a niche thing but I think it would be better to call this sporeParticles. Up to you if you want to keep it though